test/e2e/suite_test.go

```diff
@@ -152,11 +152,6 @@ func beforeTest(t *testing.T, namespace, fixtureDir string, f *framework.E2eFram
 		return err
 	}
 
-	if err := f.DeployCassOperator(namespace); err != nil {
```

Comment:
Is this because cass-operator is deployed when k8ssandra-operator is deployed? If so this supersedes #220.

Reply:
Yeah